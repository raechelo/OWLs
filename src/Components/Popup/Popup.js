import React from 'react';

function Popup(props) {

  return (
    <div className="info-popup">
      <h3>How to Play:</h3>
      <p>You can choose from the filters to quiz yourself on specific topics or run through them at random.</p>
      <p>Should you get a question correct, you will earn your house points. Should you get a quesion wrong, you will earn all other houses points.</p>
      <section className="disabled">
        <p>To study previously incorrect questions click 'Study' then Start!</p>
        <button onClick={props.studyWrongQs}>Study</button>
        <button onClick={props.clearStorage}>Clear</button>
      </section>
      <button onClick={props.randomizeQuestions}>Start!</button>
    </div>
  )
}

export default Popup;