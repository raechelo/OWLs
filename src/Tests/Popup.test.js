import React from 'react';
import Popup from '../Components/Popup/Popup';
import { shallow } from 'enzyme';

describe('Popup', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow (
      <Popup />
    )
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot()
  });
})