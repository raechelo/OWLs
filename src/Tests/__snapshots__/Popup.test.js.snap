// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup should match the snapshot with all the data passed in 1`] = `
<div
  className="info-popup"
>
  <h3>
    Ordinary [React] Wizarding Level
  </h3>
  <h3>
    How to Play:
  </h3>
  <p>
    You can choose from the filters to quiz yourself on specific topics or run through them at random.
  </p>
  <p>
    Should you get a question correct, you will earn your house points. Should you get a quesion wrong, you will earn all other houses points.
  </p>
  <section
    className="disabled"
  >
    <p>
      To Study previously incorrect questions click 'Study' then Start!
    </p>
    <button>
      Study
    </button>
    <button>
      Clear
    </button>
  </section>
  <button>
    Start!
  </button>
</div>
`;
